pkg/playground/react-demo-cards.jsx

```diff
@@ -0,0 +1,56 @@
+/*
+ * This file is part of Cockpit.
+ *
+ * Copyright (C) 2019 Red Hat, Inc.
+ *
+ * Cockpit is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation; either version 2.1 of the License, or
+ * (at your option) any later version.
+ *
+ * Cockpit is distributed in the hope that it will be useful, but
+ * WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with Cockpit; If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import React from "react";
+import ReactDOM from "react-dom";
+import "./react-demo.css";
+
+import { Button, Card, CardHeader, CardBody, CardFooter, CardHead, CardActions } from '@patternfly/react-core';
+
+const CardsDemo = () => {
+    return (
+        <div id="react-cards">
```

Comment:
Maybe it's better to use a gallery layout (basically a css grid layout)  and put the cards inside there, https://patternfly.github.io/v4/documentation/react/layouts/gallery/

It's very simple in usage - I am already using it for the new overview page.

Reply:
Cool i'll use that :)

Reply:
Based on the screenshots (I didn't build myself yet), it looks like using the Gallery is both easier and also yields better results. Let's just use that then, what do you think?